<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BSPzipGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelBSP">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>30</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select TF2 bin directory</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBSP">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>60</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelERR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMAP">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>110</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select map path</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDIR">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>190</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select output directory</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonMAP">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>140</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDIR">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>220</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonREPACK">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Repack</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCREDIT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CreditText</string>
    </property>
   </widget>
   <zorder>ButtonDIR</zorder>
   <zorder>ButtonMAP</zorder>
   <zorder>ButtonBSP</zorder>
   <zorder>labelBSP</zorder>
   <zorder>labelMAP</zorder>
   <zorder>labelDIR</zorder>
   <zorder>ButtonREPACK</zorder>
   <zorder>labelERR</zorder>
   <zorder>labelCREDIT</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
